net/gensync/Portfile

```diff
@@ -15,7 +22,7 @@ description             Gensync: a library for efficient synchronization of data
 long_description        Gensync is a library that uses many different syncing \
                         algorithms to sync data between two nodes in a network. \
                         These algorithms include IBLTs, CPISyncs, HashSyncs, Cuckoo Syncs, and more.
-
+set includedir ${prefix}
```

Comment:
Why is this line needed? I see no other mention of this `includedir` variable in this Portfile.

Reply:
Here, you are setting a Tcl variable. Doing this does not pass it anywhere as an environment variable.

Reply:
I see that your CMakeLists.txt is trying to use a variable of that name [this way](https://github.com/nislab/gensync-macports/blob/0897f0f18765068fd8c54238de77fbc426b3051f/CMakeLists.txt#L124C1-L132):

```cmake
# Check if the "includedir" variable is already set
if(NOT DEFINED includedir)
    set(includedir "/opt/local" CACHE PATH "Path to the include directory")
endif()


# Users may need to update these paths
include_directories(${includedir}/include)
link_directories(${includedir}/lib)
link_directories(${includedir}/lib64)
```

So the variable is misnamed: it's not the include directory; it's the prefix, in which the include and lib directories exist. CMake already has a variable for that, which MacPorts sets for you: [`CMAKE_INSTALL_PREFIX`](https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html). You should use that rather than invent new variables. I am not certain but you may not even need to set these include and link directories; they might already be set for you based on the prefix.

You can definitely delete the lib64 line. There never has been and probably never will be any lib64 directories on macOS.